exchanges/cryptodotcom/cryptodotcom_test.go

```diff
@@ -0,0 +1,929 @@
+	var resp *OTCTrade
+	err := json.Unmarshal([]byte(getOtcUserResponseJSON), &resp)
+	if err != nil {
+		t.Fatal(err)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, cr)
+	_, err = cr.GetOTCUser(context.Background())
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetOTCInstruments(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, cr)
+	_, err := cr.GetOTCInstruments(context.Background())
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+const requestOTCQuoteResponseJSON = `{"quote_id": "2412548678404715041", "quote_status": "ACTIVE", "quote_direction": "BUY", "base_currency": "BTC", "quote_currency": "USDT", "base_currency_size": null, "quote_currency_size": "100000.00", "quote_buy": "39708.24", "quote_buy_quantity": "2.51836898", "quote_buy_value": "100000.00", "quote_sell": "39677.18", "quote_sell_quantity": "2.52034040", "quote_sell_value": "100000.00", "quote_duration": 2, "quote_time": 1649736353489, "quote_expiry_time": 1649736363578 }`
+
+func TestRequestOTCQuote(t *testing.T) {
+	t.Parallel()
+	var resp *OTCQuoteResponse
+	err := json.Unmarshal([]byte(requestOTCQuoteResponseJSON), &resp)
+	if err != nil {
+		t.Fatal(err)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, cr)
+	_, err = cr.RequestOTCQuote(context.Background(), currency.NewPair(currency.BTC, currency.USDT), .001, 232, "BUY")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+const acceptOTCQuoteResponseJSON = `{"quote_id": "2412548678404715041", "quote_status": "FILLED", "quote_direction": "BUY", "base_currency": "BTC", "quote_currency": "USDT", "base_currency_size": null, "quote_currency_size": "100000.00", "quote_buy": "39708.24", "quote_sell": null, "quote_duration": 2, "quote_time": 1649743710146, "quote_expiry_time": 1649743720231, "trade_direction": "BUY", "trade_price": "39708.24", "trade_quantity": "2.51836898", "trade_value": "100000.00", "trade_time": 1649743718963 }`
+
+func TestAcceptOTCQuote(t *testing.T) {
+	t.Parallel()
+	var resp *AcceptQuoteResponse
+	err := json.Unmarshal([]byte(acceptOTCQuoteResponseJSON), &resp)
+	if err != nil {
+		t.Fatal(err)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, cr)
+	_, err = cr.AcceptOTCQuote(context.Background(), "12323123", "")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+const getOTCQuoteHistoryResponseJSON = `{"count": 1, "quote_list": [ { "quote_id": "2412795526826582752", "quote_status": "EXPIRED", "quote_direction": "BUY", "base_currency": "BTC", "quote_currency": "USDT", "base_currency_size": null, "quote_currency_size": "100000.00", "quote_buy": "39708.24", "quote_sell": null, "quote_duration": 2, "quote_time": 1649743710146, "quote_expiry_time": 1649743720231, "trade_direction": null, "trade_price": null, "trade_quantity": null, "trade_value": null, "trade_time": null } ] }`
+
+func TestGetOTCQuoteHistory(t *testing.T) {
+	t.Parallel()
+	var resp *QuoteHistoryResponse
+	err := json.Unmarshal([]byte(getOTCQuoteHistoryResponseJSON), &resp)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
--- FAIL: TestGetOTCQuoteHistory (0.00s)
    cryptodotcom_test.go:425: timestamp information of type <nil> is not supported
FAIL